rules/concepts.md

```diff
@@ -194,8 +198,504 @@ Available types of conditions include:
+
+Examples for presence detection include the [iCloud Binding](https://www.openhab.org/addons/bindings/icloud/#icloud-rules), [Network Binding](https://www.openhab.org/addons/bindings/network/#presence-detection-configure-target-device), [GPSTracker Binding](https://www.openhab.org/addons/bindings/gpstracker/#distance-channel-and-presence-switch), etc., but you could also use a simple wall switch.
+
+:::: tabs
+
+::: tab UI Rule
+
+![](./images/concepts/rule-leaving.jpeg)
+
+The code:
+
+```yaml
+configuration: {}
+triggers:
+  - id: "1"
+    configuration:
+      itemName: Presence
+      command: OFF
+    type: core.ItemCommandTrigger
+conditions: []
+actions:
+  - inputs: {}
+    id: "2"
+    configuration:
+      itemName: gLights
+      command: OFF
+    type: core.ItemCommandAction
+  - inputs: {}
+    id: "3"
+    configuration:
+      itemName: gThermostat
+      command: DECREASE
+    type: core.ItemCommandAction
+```
+
+:::
+
+::: tab DSL Rule
+
+```java
+rule "Turn off the lights & adjust temperature on leaving"
+when
+  Item Presence received command OFF
+then
+  gLights.sendCommand(OFF)
+  gThermostat.sendCommand(DECREASE)
+end
+```
+
+:::
+
+::: tab JS Scripting
+
+JS Scripting
+
+```javascript
+const { rules, triggers, items } = require('openhab');
+
+rules.JSRule({
+  name: 'Turn off the lights & adjust temperature on leaving',
+  description: 'When you leave home, turn off the lights and lower the temperature',
+  triggers: [
+    triggers.ItemCommandTrigger('Presence', 'OFF') // Triggers when Item Presence is commanded OFF
+  ],
+  execute: (event) => {
+      items.getItem('gLights').sendCommand('OFF');
+      items.getItem('gTheromstat').sendCommand('DECREASE');
+  }
+});
+```
+
+:::
+
+::::
+
+### When you return home, and it is between 1 PM and 6 PM, play your favorite music
+
+:::: tabs
+
+::: tab UI Rule
+
+![](./images/concepts/rule-arrival.jpeg)
+
+The code:
+
+```yaml
+configuration: {}
+triggers:
+  - id: "1"
+    configuration:
+      itemName: Presence
+      command: ON
+    type: core.ItemCommandTrigger
+conditions:
+  - inputs: {}
+    id: "3"
+    configuration:
+      startTime: 13:00
+      endTime: 18:00
+    type: core.TimeOfDayCondition
+actions:
+  - inputs: {}
+    id: "2"
+    configuration:
+      itemName: Soundbar
+      command: ON
+    type: core.ItemCommandAction
+```
+
+:::
+
+::: tab JS Scripting
+
+JS Scripting
+
+```javascript
+const { rules, triggers, items, time } = require('openhab');
+
+rules.JSRule({
+  name: 'Play music on arrival, but only on afternoon',
+  description: 'When you return home, and it is between 1 PM and 6 PM, play your favorite music',
+  triggers: [
+    triggers.ItemCommandTrigger('Presence', 'ON') // Triggers when Item Presence is commanded ON
+  ],
+  execute: (event) => {
+      if (time.toZDT().isBetweenTimes('01:00 PM', '06:00 PM')) {
+          items.getItem('Soundbar').sendCommand('ON');
+      }
+  }
+});
+```
+
+:::
+
```

Comment:
::: tab jRuby

jRuby

```
require 'openhab'

rule 'Play music on arrival, but only on afternoon' do
  received_command Presence, command: on
  run { 
    Soundbar >> on
  }
  only_if { TimeOfDay.now.between? ‘1pm’..’6pm’
end
```
:::


Note: for the JSRule example, I'm not sure that will work. 01:00 PM doesn't really make sense. It'd be either `1:00 PM` or `13:00`. The `01` would only be used for 24 hour time, not AM/PM time.

Reply:
Thanks for the JS tip, I am not familiar with AM/PM since we use 24 hour time in Germany.